backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/database/service/DBService.java

```diff
@@ -100,6 +121,70 @@ public Optional<String> loadTargetLawXmlByEli(LoadTargetLawXmlPort.Command comma
     return targetLawRepository.findByEli(command.eli()).map(TargetLawDto::getXml);
   }
 
+  @Override
+  public Optional<Norm> loadNorm(LoadNormPort.Command command) {
+    return normRepository.findByEli(command.eli()).map(NormMapper::mapToDomain);
+  }
+
+  @Override
+  public Optional<Announcement> loadAnnouncement(LoadAnnouncementPort.Command command) {
+    return announcementRepository
+        .findByNormDtoEli(command.eli())
+        .map(AnnouncementMapper::mapToDomain);
+  }
+
+  @Override
+  public List<Announcement> loadAllAnnouncements() {
+    return announcementRepository.findAll().stream()
+        .map(AnnouncementMapper::mapToDomain)
+        .sorted(
+            Comparator.comparing(
+                    (Announcement announcement) ->
+                        announcement.getNorm().getPublicationDate().orElse(LocalDate.MIN))
```

Comment:
i added this here as the existing loading for amending laws also does this sort, i'm a bit unsure if we should do this at this layer